cumulus_library/actions/builder.py

```diff
@@ -94,7 +95,15 @@ def _load_and_execute_builder(
     table_builder_class = table_builder_subclasses[0]
     table_builder = table_builder_class()
     if write_reference_sql:
```

Comment:
nit: this code path is starting to feel like it should call out to a separate function, as it grows logic like this. Not necessary now, just saying.

Reply:
yeah hopefuly this is the last time we touch it like this.